### ISSUE TITLE
Styled-components comment spacing

### ISSUE THREAD

**Issue by dreamorosi** [gandm/language-babel#303]
Hi, thanks for making the package I use it a lot!

I use styled-components and when commenting out one or more lines the closing part of the comment doesn't add a space at the end of the line like shown above.
![comment](https://cloud.githubusercontent.com/assets/7353869/21425969/97a39186-c84c-11e6-8d10-d23a014d552f.gif)

Would you consider adding the space? This not only would avoid linting warnings but also favour consistency as it's present at the beginning of the comment.

So in short the comment would go from
`${''/* transform: translate(0, -180px)*/}`
to
`${''/* transform: translate(0, -180px) */}`

I've read on the [README.md](https://github.com/gandm/language-babel/commit/552261cef7c3735382bf162ead540cf7052e6c5c#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR2) that you're ceasing support so I was wondering if a PR with the fix would be welcome.

**Comment by gandm** [comment]
Just added a commit to fix this. I will publish when Atom 1.13 comes out of beta.

**Comment by dreamorosi** [comment]
Awesome, thanks!

**Comment by gandm** [comment]
Published in 2.49.4
